symfit/core/models.py

```diff
@@ -1109,7 +1127,9 @@ def eval_components(self, *args, **kwargs):
             initial_dependent,
             t_smaller,
             args=tuple(
-                bound_arguments.arguments[param.name] for param in self.params),
+                bound_arguments.arguments[param.name]
+                for param in self.model_params
```

Comment:
Personally I see this as an example of PEP8-ing gone mad. It's more readable if this was on one line, and the offense is not so big so I prefer it.